django-stubs/utils/tree.pyi

```diff
@@ -14,10 +14,14 @@ class Node:
     def __init__(
         self, children: _NodeChildren | None = ..., connector: str | None = ..., negated: bool = ...
     ) -> None: ...
+    @classmethod
+    def create(cls, children: _NodeChildren | None = ..., connector: str | None = ..., negated: bool = ...) -> Node: ...
+    def copy(self) -> Node: ...
```

Comment:
Yes, this is `Node`: https://github.com/django/django/blob/fcc5091b4a7c29e2912f6c8b2e0ebc64f4853457/django/utils/tree.py#L36 👍